modules/openapi-generator/src/main/resources/Java/pojo.mustache

```diff
@@ -11,7 +11,7 @@
 @JsonTypeName("{{name}}")
 {{/jackson}}
 {{>additionalModelTypeAnnotations}}{{>generatedAnnotation}}{{#discriminator}}{{>typeInfoAnnotation}}{{/discriminator}}{{>xmlAnnotation}}
-public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#vendorExtensions.x-implements}}{{#-first}}implements {{{.}}}{{/-first}}{{^-first}}, {{{.}}}{{/-first}}{{#-last}} {{/-last}}{{/vendorExtensions.x-implements}}{
+public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#vendorExtensions.x-implements}}{{#-first}}implements {{{.}}}{{/-first}}{{^-first}}, {{{.}}}{{/-first}}{{#-last}} {{/-last}}{{/vendorExtensions.x-implements}}{{#parcelableModel}}implements Parcelable {{/parcelableModel}}{
```

Comment:
I think you will need to update `{{vendorExtensions.x-implements}}` (in the Java client generator class) to include `implements Parcelable`

Let me know if you need help on that.

Reply:
I'll give it a look, I'll let you know if I do, thanks!